addons/account/models/account_move.py

```diff
@@ -1234,7 +1234,20 @@ def _compute_tax_totals(self):
                             handle_price_include=False,
                             extra_context={'_extra_grouping_key_': 'epd'},
                         ))
-                move.tax_totals = self.env['account.tax']._prepare_tax_totals(**kwargs)
+
+                amount_untaxed = abs(move.amount_untaxed_signed)
+                amount_total = abs(move.amount_total_signed)
+                currency = move.currency_id or move.journal_id.currency_id or move.company_id.currency_id
+                if move.state == 'posted':
```

Comment:
Why "posted"? We can have discrepancies with the journal entry created even if it is in draft, right (and then we post it keeping the same state)?